src/queue.c

```diff
@@ -976,6 +976,7 @@ _dispatch_get_mach_host_port(void)
 #include <unistd.h>
 #include <sys/syscall.h>
 
+#ifndef __ANDROID__
```

Comment:
@MadCoder `gettid` is already defined in `unistd.h:57` for API 21 (and lower also) as `extern pid_t  gettid(void) __pure2`. Android declaration is not `inline` as `src/queue.c`, but it shouldn’t have a performance impact in this context, right?